apps/crypto-bot/crypto-bot.yml

```diff
@@ -0,0 +1,12 @@
+name: Crypto-Bot
+description: 'app for encryption and decryption'
```

Comment:
No need to put "app" in the description. Also is it for encrypting files? Messages across a network?

Reply:
no network request. It works locally

Reply:
Got it. Please update the description so this is clear to everyone.